.github/conversational/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+* **Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines
+- [ ] Unit Tests for the changes that have been added (for bug fixes / features)
```

Comment:
We don't have unit tests for the repo as well. Instead ask to post a screenshot of your change